src/js/components/sharedComponents/QuarterPickerWithFY.jsx

```diff
@@ -1,12 +1,21 @@
-import React, { useState, useEffect, useCallback, useRef } from 'react';
+/**
+ * QuarterPickerWithFY
+ * Created by Max Kendall 10/25/2020
+ * TODOs:
+ * 1. this component is coupled to the fy/period logic for accounts. 😰
```

Comment:
I am not sure if I understand what this todo is stating, can we change it to something like `decouple this component......`?

Reply:
sure